src/client.js

```diff
@@ -302,6 +303,58 @@ class Client {
     return Realms.decode(rawResponse)
   }
 
+  /**
+   * Requests the deletion of a named TozID Realm belonging to the account.
+   *
+   * @param {string} realmName The name for the realm to delete.
+   *
+   * @returns {Promise<Object>} Empty object.
+   */
+  async deleteRealm(realmName) {
+    return this.api.deleteRealm(this.queenClient, realmName)
```

Comment:
What happens if a user passes account as the realmName?

Reply:
BTW I'm still (re)learning the GitHub UI, I just want to leave an inline comment without approving or requiring changes...

Reply:
i'm not sure i follow. passing anything to this will result in a DELETE request to `/v1/identity/realm/${what-they-pass-in}`. if it is not a string or it is a string that doesn't correspond to a real realm, it will error.

is there a concern i'm unaware of here?

additionally, the only change i made to this particular method was to colocate it with the other realm-specific functions. did you discover a new bug!? 😀

Reply:
ok I didn't realize this was pre-existing code - I missed the delete. I haven't used this api yet and I was curious if you could delete the top level account

Reply:
gotcha! not yet. but i think @rosasck is adding the requisite API pieces to the server so we can in the future this sprint. :shipit: